lib/tokenizeCommandString.js

```diff
@@ -1,5 +1,26 @@
 /* global tokenizeCommandString:true */
 /* exported tokenizeCommandString */
 
-// we need to filter out non-args in case the command string was `/cycle `
-tokenizeCommandString = str => str.split(/\s+/).filter(arg => arg.trim().length > 0)
+tokenizeCommandString = commandStr => {
+  let inSingleQuotes = false
+  let inDoubleQuotes = false
+  const commandArray = commandStr.split('')
+  // replace argument-delimiters with '\n' which we'll use to split-on later
+  for (let i = 0; i < commandArray.length; ++i) {
+    if (commandArray[i] === "'" && !inDoubleQuotes) {
+      inSingleQuotes = !inSingleQuotes
+      commandArray[i] = '\n'
+    }
+    if (commandArray[i] === '"' && !inSingleQuotes) {
+      inDoubleQuotes = !inDoubleQuotes
+      commandArray[i] = '\n'
+    }
+    if (commandArray[i] === ' ' && !inSingleQuotes && !inDoubleQuotes) {
+      commandArray[i] = '\n'
+    }
+  }
+  return commandArray
+    .join('')                        // turn array back into string
+    .split('\n')                     // split the arguments into an array
+    .filter(arg => arg.length > 0)   // empty arguments aren't arguments
+}
```

Comment:
We may have to reach for a regex here. This PCRE one will do the job (https://regex101.com/r/rF7aF5/2), but I'm not sure if there's a good way to do that without named captured (which I gues JS doesn't support?)